### PR TITLE
Made constants fully upper case in camera server

### LINKED ISSUE #29857: CameraServer has strange enum values
**Godot version:**
3.2 master
Zrzut ekranu z 2019-06-17 22-41-26

**Issue description:**
It is strange that 3 values are equal 0 in one enum

### PR DESCRIPTION
**Pull request by BastiaanOlij**
Just making sure all constants in camera server are fully upper case.
Have to remember to check ARKit and ARCore PRs once this is merged. 

*Bugsquad edit:* Fixes #29857.

### CODE REVIEW

File: doc/classes/CameraFeed.xml [hunk 1]
```diff
@@ -45,10 +45,10 @@
 		<constant name="FEED_RGB" value="1" enum="FeedDataType">
 			Feed supplies RGB images.
 		</constant>
-		<constant name="FEED_YCbCr" value="2" enum="FeedDataType">
+		<constant name="FEED_YCBCR" value="2" enum="FeedDataType">
 			Feed supplies YCbCr images that need to be converted to RGB.
 		</constant>
-		<constant name="FEED_YCbCr_Sep" value="3" enum="FeedDataType">
+		<constant name="FEED_YCBCR_Sep" value="3" enum="FeedDataType">
```
Comment (akien-mga) [inline doc/classes/CameraFeed.xml]: "Sep" is not uppercase yet

Comment (akien-mga) [reply doc/classes/CameraFeed.xml]: (Note: stopped CI since I know you'll have to amend, don't mind the failed checks.)

Comment (BastiaanOlij) [reply doc/classes/CameraFeed.xml]: Already noticed and fixed it :)

**Review by akien-mga — APPROVED** [review]

**Comment by akien-mga** [comment]
Thanks!

**Pull request merged**